python/src/uagents/network.py

```diff
@@ -150,6 +150,17 @@ class AlmanacContract(LedgerContract):
 
     """
 
+    def get_contract_version(self) -> str:
+        """
+        Get the version of the contract.
+
+        Returns:
+            str: The version of the contract.
+        """
+        query_msg = {"query_contract_state": {}}
+        response = self.query(query_msg)
+        return response["contract_version"]
```

Comment:
I feel like this can lead to errors if the contract response is faulty and that we need to add a bit more checks in `self.query()` otherwise it looks good!